packages/shared/src/utils/cfd/cfd.js

```diff
@@ -157,40 +157,72 @@ export const isLandingCompanyEnabled = ({ landing_companies, platform, type }) =
     return false;
 };
 
-export const getIdentityStatusInfo = account_status => {
-    const poa_status = account_status?.authentication?.document?.status;
+export const getAuthenticationStatusInfo = account_status => {
     const poi_status = account_status?.authentication?.identity?.status;
+    const poa_status = account_status?.authentication?.document?.status;
 
     const idv_status = account_status?.authentication?.identity?.services?.idv?.status;
     const onfido_status = account_status?.authentication?.identity?.services?.onfido?.status;
     const manual_status = account_status?.authentication?.identity?.services?.manual?.status;
 
     const acknowledged_status = ['pending', 'verified'];
+    const failed_cases = ['rejected', 'expired', 'suspected'];
 
+    const poa_not_submitted = poa_status === 'none';
+    const poa_acknowledged = acknowledged_status.includes(poa_status);
+    const need_poa_submission = !poa_acknowledged;
+    const need_poa_resubmission = failed_cases.includes(poa_status);
+    const poa_verified = poa_status === 'verified';
+
+    const poi_not_submitted = poi_status === 'none';
+    const poi_or_poa_not_submitted = poa_not_submitted || poi_not_submitted;
+
+    //vanuatu
+
+    const poi_verified_for_vanuatu = onfido_status === 'verified' || manual_status === 'verified';
     const poi_acknowledged_for_vanuatu =
         (onfido_status && acknowledged_status.includes(onfido_status)) ||
         (manual_status && acknowledged_status.includes(manual_status));
 
-    const need_poi_for_vanuatu = !poi_acknowledged_for_vanuatu;
+    const poi_pending_for_vanuatu =
+        ((onfido_status && onfido_status === 'pending') || (manual_status && manual_status === 'pending')) &&
```

Comment:
@amina-deriv is it gonna work the same way 😊?
```suggestion
        ((onfido_status === 'pending') || (manual_status === 'pending')) &&
```